captum/insights/frontend/src/utils/color.ts

```diff
@@ -12,7 +12,7 @@ function calcHSLFromScore(percentage, zeroDefault = false) {
   const default_hsl = [0, 40, zeroDefault ? 100 : 90];
   const abs_percent = Math.abs(percentage * 0.01);
   if (abs_percent < 0.02) {
-    return default_hsl;
+    return `hsl(${default_hsl[0]}, ${default_hsl[1]}, ${default_hsl[2]})`;
```

Comment:
thanks for fixing this. I think it would be simpler to do:

```js
let color = default_hsl;
if abs_percent > 0.02 {
  color = [....];
}
return `hsl(...)`;

Reply:
The last two values in hsl need `%` sign, fixed it.